helm/alfresco-content-services/templates/config-repository.yaml

```diff
@@ -53,13 +53,6 @@ data:
       {{- if .Values.transformmisc.enabled }}
       -DlocalTransform.misc.url=http://{{ template "content-services.shortname" . }}-transform-misc
       {{- end }}
-      -Dalfresco-pdf-renderer.url=http://{{ template "content-services.shortname" . }}-pdfrenderer
```

Comment:
This file is shared across all deployments so we'll have to find a way to optionally include/exclude these properties based on repo version.

For now, if they cause no harm being there in 7.0 let's just leave them there for now, but please add a comment to remind us to address it.

Reply:
Agreed. Restored